database/insertDatabase.js

```diff
@@ -8,12 +8,22 @@ const db = mongoose.connection;
 db.on('error', console.error.bind(console, 'connection error:'));
 
 
-for (let i = 0; i < 100; i += 1) {
+const randomize = function() {
+  const ary = [];
+  const rand = Math.floor(Math.random() * 11);
+  for (var i = 0; i < rand; i+=1) {
+    const randomNum = Math.floor(Math.random() * 100) - 1;
+    ary.push(`https://imagebasket.s3-us-west-1.amazonaws.com/image${randomNum}.jpg`);
+  }
+  return ary;
+}
+
+for (let i = 0; i < 5; i += 1) {
 
   const test = new imageData.images ({
     id: i,
     name: faker.commerce.productName(),
-    url: `https://imagebasket.s3-us-west-1.amazonaws.com/image${i}.jpg`,
+    url: randomize().shift(`https://imagebasket.s3-us-west-1.amazonaws.com/image${i}.jpg`),
   });
 
   test.save( (err, test) => {
```

Comment:
Be sure to check your database and make sure each object created by the for loop is getting more than one image inside the array stored at the url property. Also check line 21 to make sure at least 100 items are created when you run the script instead of 5.

Reply:
Got it

Reply:
and it is unshift() instead of shift()